### ISSUE TITLE
Fix device call to pow for CUDA + C++11 + GCC

### ISSUE THREAD

**Issue by Flamefire** [ComputationalRadiationPhysics/picongpu#1010]
CUDA with GCC host compiler and C++11 enabled cannot use the device side pow(float, int) and pow(double, int) functions: http://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html#cuda-tools-title-known

Current workaround (https://github.com/ComputationalRadiationPhysics/picongpu/pull/1005) casts the int arguments to float/double which results in reduced performance.

Probably an own version of the int-exponent functions should be put together (square-and-multiply) should be implemented until a fix by NVIDIA is out and pow can be used independently of host or device side.

**Comment by ax3l** [comment]
@Flamefire did  #1005 close this issue?

**Comment by Flamefire** [comment]
@ax3l No it does not. I think you answered your own question with the quote ;)
We got to wait for a fix by nvidia till this can be closed

**Comment by ax3l** [comment]
yes, found it again and cross-referenced it with an update yesterday ^^
